pkg/aggregator/demultiplexer.go

```diff
@@ -196,17 +207,31 @@ func initAgentDemultiplexer(options DemultiplexerOptions, hostname string) *Agen
 // Run runs all demultiplexer parts
 func (d *AgentDemultiplexer) Run() {
 	if !d.options.DontStartForwarders {
+		// orchestrator forwarder
```

Comment:
could you move this comment after the next line? 🙇

Reply:
yes this was a bad conflict resolution